<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Connect</class>
 <widget class="QWidget" name="Connect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connect</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>服务器Ip</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="editIP">
     <property name="text">
      <string>192.168.3.15</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>图片端口</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="editImg">
     <property name="text">
      <string>2002</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>数据端口</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="editData">
     <property name="text">
      <string>3000</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>相机参数1</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>[x0,y0]</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinx0">
        <property name="maximum">
         <number>800</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QSpinBox" name="spiny0">
        <property name="maximum">
         <number>800</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>[dx,dy]</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spindx">
        <property name="maximum">
         <number>1600</number>
        </property>
        <property name="value">
         <number>1600</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="spindy">
        <property name="maximum">
         <number>2000</number>
        </property>
        <property name="value">
         <number>1200</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>[incrx,incry]</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="spinix">
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QSpinBox" name="spiniy">
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="buttonstart">
        <property name="text">
         <string>开始相机</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="comboBox_2">
        <item>
         <property name="text">
          <string>彩色图像</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>灰色图像</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>.停止数据</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>相机参数2</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QPushButton" name="btExposure">
        <property name="text">
         <string>曝光值</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinExposure">
        <property name="maximum">
         <number>30000</number>
        </property>
        <property name="value">
         <number>3000</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="spinGain">
        <property name="maximum">
         <number>2000</number>
        </property>
        <property name="value">
         <number>200</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="btGain">
        <property name="text">
         <string>增益</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>触发模式</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <item>
         <property name="text">
          <string>TRGSRC_UNSET 0 </string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TRGSRC_IMM    1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TRGSRC_EXT      2</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="buttonstop">
     <property name="text">
      <string>复位</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
